classes/Pods.php

```diff
@@ -3736,7 +3736,7 @@ public function form( $params = null, $label = null, $thank_you = null ) {
 				'name' => $name,
 			];
 
-			if ( ! is_array( $field ) ) {
+			if ( ! is_array( $field ) && ! is_object( $field ) ) {
```

Comment:
```suggestion
			if ( is_string( $field ) ) {
```

Reply:
This might be better in hindsight, thoughts? @unknownnf

Reply:
I've updated the commit, checked with everything too. The only cases where this would fail would be empty arrays I guess? But that should not be the case at this point.